pkgs/games/openrct2/engine.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv, fetchurl, cmake, pkgconfig
+, SDL2, SDL2_ttf, fontconfig, jansson, speexdsp, openssl, curl }:
```

Comment:
@janisozaur Hmm, the build instructions claim that it's [an (optional) dependency](https://github.com/OpenRCT2/OpenRCT2/wiki/Building-OpenRCT2-on-Linux). Are those instructions outdated?